Blockzilla/OverlayView.swift

```diff
@@ -9,24 +10,30 @@ protocol OverlayViewDelegate: class {
     func overlayViewDidTouchEmptyArea(_ overlayView: OverlayView)
     func overlayViewDidPressSettings(_ overlayView: OverlayView)
     func overlayView(_ overlayView: OverlayView, didSearchForQuery query: String)
+    func overlayView(_ overlayView: OverlayView, didSubmitText text: String)
 }
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-
+    //Settings Button
```

Comment:
My theory on comments: "Comments should be focused on _why_ the code is doing what it's doing, not the _what_." I _think_ this is a what comment :) .